bin/upgrade.bash

```diff
@@ -93,6 +113,28 @@ apply() {
 
   snippets_check apply "${snippets}"
 
+  local prepared_version
+  if [[ "${CK8S_CLUSTER:-}" =~ ^(sc|both)$ ]]; then
+    prepared_version="$(get_prepared_version sc || true)"
+    if [ -z "${prepared_version}" ]; then
+      log_fatal "'prepare' step does not appear to have been run, do so first"
+    fi
+
+    if [[ "${prepared_version}" != "${CK8S_TARGET_VERSION}" ]]; then
+      log_fatal "'prepare' step in sc appears to have been run for version ${prepared_version}, not ${CK8S_TARGET_VERSION}"
+    fi
```

Comment:
suggestion: Extract into a separate function and reuse for both sc and wc.

Reply:
1877e26f